public/assets/styles/main.scss

```diff
@@ -33,7 +35,7 @@ a {
 #player-main-artist span, #player-main-title span, #player-main-album span{
   background: $black-grey;
   color: $light-grey;
-  padding: 2px 5px;
+  padding: 2px 7px;
```

Comment:
the artist mouseover is a pointer but the artist title itself is an 'auto' cursror